app/src/main/res/values/strings.xml

```diff
@@ -790,4 +790,7 @@
     <string name="audio_track_type_original">original</string>
     <string name="audio_track_type_dubbed">dubbed</string>
     <string name="audio_track_type_descriptive">descriptive</string>
+    <string name="show_playlist_duration_key" translatable="false">show_playlist_duration</string>
+    <string name="show_playlist_duration_summary" translatable="false">Show overall duration in playlists, for larger playlists it might not display the actual duration</string>
+    <string name="show_playlist_duration_title" translatable="false">Show playlist duration</string>
```

Comment:
We'll need to move the key into settings_keys.xml.
```suggestion
    <string name="show_playlist_duration_title">Show playlist duration</string>
    <string name="show_playlist_duration_summary">Show combined duration of all streams in playlists. The display duration might be too small if the playlist has many items.</string>
```

Reply:
There is no setting anymore so I don't know why this is still here ;-)